web/packages/teleterm/src/ui/Search/SearchContext.tsx

```diff
@@ -76,16 +80,21 @@ export const SearchContextProvider: FC = props => {
   const close = useCallback(() => {
     setIsOpen(false);
     setActivePicker(actionPicker);
-    if (previouslyActive.current instanceof HTMLElement) {
```

Comment:
AFAIK `previouslyActive` can be for example an `SVGElement` that doesn't have focus method, so we want to run it only for `HTMLElement`. That's why I checked an instance of the element.  
https://stackoverflow.com/a/47198823

I think you can still have that simple mock in a test, but only revert the `instanceof` check?

Reply:
`SVGElement` extends `HTMLOrSVGElement` which has `focus`, though I've seen people mention what you said (https://github.com/Microsoft/TypeScript/issues/5901#issuecomment-161716750), but I suppose it has changed since 2015.

I cannot use the `instanceof` check together with a simple mock because the `instanceof` check is done in runtime and the object is not an instance of `HTMLElement`.

[MDN doesn't explain](https://developer.mozilla.org/en-US/docs/Web/API/Document/activeElement) how it's possible for `document.activeElement` to be an `Element` that doesn't have `focus` function. I assume this is just due to the internal browser engine hierarchy of DOM objects.

For now, I'll revert `previouslyActive` back to just `Element` to adhere to the browser API. Before calling `focus` I'll check if `previouslyActive` has the `focus` property which is a function. I think this is better than an `instanceof` check because after all we're not interested in the element being an instance of a specific class, we just want to call `focus` on it.

Reply:
I have to use a less type-safe check anyway because the current code won't pass CI until #25683 gets merged and I don't want to be blocked by it. I'll add a TODO comment to address it later.